### PR TITLE
Update logging for ATO

### PR DESCRIPTION
**Pull request by rogeruiz**
This body of work includes the updated logging Tock needs for the ATO.

You can test this by going to https://tock-logging.app.cloud.gov

You can find the logs in Kibana at https://logs.fr.cloud.gov and search for `@cf.app:"tock-logging" AND @message:"[...]"` with `[...]` being messaging from the #752 docs around logging.

### List of logging needed by the ATO
- [x] successful and unsuccessful account logon events
- [x] account management events
- [x] object access
- [x] policy change
- [x] privilege functions
- [x] process tracking
- [x] system events
- [x] all administrator activity
- [x] authentication checks
- [x] authorization checks
- [x] data deletions
- [x] data access
- [x] data changes
- [x] permission changes

### CODE REVIEW

File: bin/run.sh [hunk 1]
```diff
@@ -17,5 +17,32 @@ else
...
+# Adding license key from script $(basename "${0}")
+api_key=${NEW_RELIC_API_KEY}
+EOF
+
+DEPLOYMENT_DESCRIPTION="Recording deployment of ${VERSION}."
+
+echo "${DEPLOYMENT_DESCRIPTION}"
+
+# Record deployment using the New Relic Python Admin CLI
+newrelic-admin record-deploy "${NEW_RELIC_CONFIG_FILE}" "${DEPLOYMENT_DESCRIPTION}"
```
Comment (ccostino) [inline bin/run.sh]: Oh, I think I see now... I had asked if this this additional New Relic setup needed or if the initialization in [`wsgi.py`](https://github.com/18F/tock/blob/master/tock/tock/wsgi.py#L16-L25) is enough, but this is being used to log the actual deployment, which is awesome if I'm understanding correctly! :-)

File: tock/tock/settings/base.py [hunk 2]
```diff
@@ -34,13 +34,13 @@
...
-    'organizations',
-    'api',
-    'utilization',
+    'tock.apps.TockAppConfig',
+    'projects.apps.ProjectsAppConfig',
+    'hours.apps.HoursAppConfig',
+    'employees.apps.EmployeesAppConfig',
+    'organizations.apps.OrganizationsAppConfig',
+    'api.apps.ApiAppConfig',
+    'utilization.apps.UtilizationAppConfig',
```
Comment (ccostino) [inline tock/tock/settings/base.py]: Oh 👏 on the better app organization!

**Comment by rogeruiz** [comment]
thanks @ccostino yeah. that's thanks to @toolness work around updating to `1.11` which suggests the `AppConfig` approach for sub-apps. 

This PR is ready to be reviewed. All the events should be logged in Kibana, which is where the accessors will be verifying that the logs are being recorded properly. Let me know if there are any logs I forgot, or if the logging doesn't make sense from a language perspective.
cc @annalee 

[Docs for ATO logging is here](https://github.com/18F/tock/blob/ato-documentation/docs/logging.md).

File: tock/employees/signals.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import logging
```
Comment (tbaxter-18f) [inline tock/employees/signals.py]: I'm not saying there's anything wrong here, but I definitely like overriding `save()` on the model more than signals. I think it's more transparent, less likely to fail silently, and avoids weird circular dependency problems. Again, not asking for any changes, just throwing it out there.

File: tock/tock/settings/base.py (hunk 2 above)
Comment (tbaxter-18f) [reply tock/tock/settings/base.py]: Given than employees, organizations, etc, are the apps, this is actually a bit confusing to my eyes.

**Comment by tbaxter-18f** [comment]
I agree with Annalee. I definitely don't want my general opinions about signals to be a blocker, and there _is_ a big advantage in keeping all of those in one place.

**Comment by rogeruiz** [comment]
@ccostino @annalee This is good to be reviewed now against the [updated ATO logging docs located here](https://github.com/18F/tock/blob/ato-documentation/docs/logging.md).

File: tock/employees/signals.py (hunk 3 above)
Comment (ccostino) [reply tock/employees/signals.py]: I'm +1 with on you @tbaxter-18f regarding the signals in general, but I have also run through this with @rogeruiz and we're good to go here!

**Pull request merged**